### PR TITLE
Frontend scaffolding

### PR DESCRIPTION
**Pull request by Kris2ffer**

### CODE REVIEW

File: FrontEnd/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "msal-react",
```
Comment (gjersvik) [inline FrontEnd/package.json]: Should not this be "sepes-frontend" or something like that?

File: FrontEnd/public/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (gjersvik) [inline FrontEnd/public/index.html]: Please call the app Sepes Data Sandbox or something similar.

File: FrontEnd/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (gjersvik) [inline FrontEnd/public/manifest.json]: Update short name and name to be Sepes related. Sepes and Sepes Data Sandbox for example.

File: FrontEnd/src/App.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
+.App {
```
Comment (gjersvik) [inline FrontEnd/src/App.css:1]: No rules from this css file is used. As far as i can see.
Remove or make empty or add some nice Sepes login logout buttons and text.

File: FrontEnd/src/App.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      console.log(error);
+    });
+}
+
+function logout() {
+  msalApp.logout();
+}
+
+function showInfo() {
+  document.getElementById("tokenName").innerText = msalApp.getAccount().name;
```
Comment (gjersvik) [inline FrontEnd/src/App.js]: This is a super no-no for React. React uses a Virtual DOM so it only updates the Browser DOM when a new version is different. And then only the parts that changes. If you updates the Browser DOM underneath React you will be fighting it every step of the way. And super wired bugs will emerge.

You most likely wants want reacts call a Stateful Component.

File: FrontEnd/src/logo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (gjersvik) [inline FrontEnd/src/logo.svg]: Cant see that this file is used.

**Review by gjersvik — CHANGES_REQUESTED** [review]

**Review by Unicus-Tom — APPROVED** [review]

File: FrontEnd/public/manifest.json [hunk 7]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Sepes",
+  "name": "CSepes Data Sandbox",
```
Comment (gjersvik) [inline FrontEnd/public/manifest.json]: Just a small typo.

**Review by gjersvik — APPROVED** [review]
All my concerns was addressed found only a small typo.

**Pull request merged**